index.js

```diff
@@ -0,0 +1,2 @@
+const nombreUsuario = prompt("ingrese su nombre");
+alert(`hola ${nombreUsuario}`);
```

Comment:
En este caso, me parece que seria mejor que no uses interpolacion  de strings.
```suggestion
alert('hola' + nombreUsuario);
```